app/controllers/content/index.js

```diff
@@ -206,6 +153,12 @@ export default Controller.extend(Analytics, {
             .replace(/\s+\S*$/, '');
     }),
 
+    emailHref: computed('model', function() {
```

Comment:
note, this is only here because this link generated is used for metrics tracking stuff.